glossary.md

```diff
@@ -6,6 +6,20 @@ category: glossary
 
 # Glossary
 
+## A
+
+### ACID
+
+ACID refers to the four key properties of a transaction: atomicity, consistency, isolation, and durability. Each of these properties is described below.
+
```

Comment:
The descriptions of the four properties are very different from the original Chinese. Why is that? Have you reminded the author of this difference? @anotherrachel

Reply:
Actually, I enrich the descriptions for the Chinese version. The original can be seen here: https://pingcap.com/blog-cn/best-practice-optimistic-transaction/
In English, however, I do refer to some references and make adjustments. It is not translated according to the Chinese as intended. But it is open to discussion. The part focuses on how ACID is achieved in TiDB.

Reply:
If there's nothing wrong with the original texts, do you think it's a better idea that we try to keep the English documentation consistent with the Chinese documentation?

My concern is that the content might be updated later, if we maintain two different versions of the same content, it might bring extra burden to the author.

But still, it's my own opinion. For this particular content, I don't think it will be updated in the foreseeable future, so it's okay to keep your enriched descriptions. (We'd better merge this PR ASAP.)